cluster/addons/node-problem-detector/npd.yaml

```diff
@@ -42,6 +42,15 @@ spec:
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
+      - name: logs
+        image: busybox
```

Comment:
Please don't use busybox. @tallclair can point you to the post about why (with the recommendations of what the alternative image we recommend is).

Reply:
I would be glad to look at this post. I didn't know that is wasn't a good practice to use busybox images. It would be useful also to find the alternative image to use.

Reply:
Why is this a separate container? Can you just tee the output in the `node-problem-detector` container?

Reply:
@visheyra like this

```diff
-         - "/node-problem-detector --logtostderr --system-log-monitors=/config/kernel-monitor.json,/config/kernel-monitor-filelog.json,/config/docker-monitor.json,/config/docker-monitor-filelog.json >>/var/log/node-problem-detector.log 2>&1"
+         - "/node-problem-detector --logtostderr --system-log-monitors=/config/kernel-monitor.json,/config/kernel-monitor-filelog.json,/config/docker-monitor.json,/config/docker-monitor-filelog.json 2>&1 | tee -a /var/log/node-problem-detector.log"
```